internal/platform/utimes_darwin.go

```diff
@@ -0,0 +1,28 @@
+package platform
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+var libc_futimens_trampoline_addr uintptr
+
+//go:cgo_import_dynamic libc_futimens futimens "/usr/lib/libSystem.B.dylib"
```

Comment:
if this is a note, maybe make a comment above and tab-indent the go:cgo_import_dynamic, if this is a real cgo import, which I think it is, see the below comment :D

Reply:
yes this is required dynamically link against libc, import `futimens` and populate `var libc_futimens_trampoline_addr` with the address of the trampoline that is defined in the companion `.s` file